lib/cogip_defs/include/cogip_defs/Pose.hpp

```diff
@@ -63,7 +63,7 @@ class Pose : public Coords {
         pose.set_O(O_);
     };
 
-private:
+protected:
```

Comment:
Why this change ?

Reply:
Because it is used if path::Pose, but it is not in the related commit.
Fixed.